### ISSUE TITLE
Tooltip not appearing/disappearing when hovering from one element to the next (without a gap) 

### ISSUE THREAD

**Issue by de-code** [romainberger/react-portal-tooltip#39]
When two elements with tooltips are next to (or on top of) each other then there won't be a gap where the tooltip is disabled. The tooltip is then not shown correctly if at all.

**Comment by Redmega** [comment]
Is this fixed by  #23?

**Comment by Redmega** [comment]
Thanks for that workaround. In my case I also had to add a timeOut on the mouseLeave as well.

**Comment by vegetableman** [comment]
Faced the same issue, just giving a different group prop for each tootltip also helps.

**Comment by Redmega** [comment]
@vegetableman The issue with doing that is that it will create multiple tooltips in the DOM. If all the tooltips are in the same group, they will reuse the same DOM element.

**Comment by vegetableman** [comment]
@Redmega Don't think that's a big deal, after all, libraries like `tippy` do the same.

**Comment by Redmega** [comment]
Fair enough @vegetableman. Just thought I'd mention it, some users need to keep DOM elements to a minimum for whatever reason.
